### PR TITLE
new Project Maintainer UI @home

### LINKED ISSUE #868: Make new Project Maintainer UI
## Requirements

- Try to have a swish effect like the cards in [this](https://www.hellomonday.com/about)
- Below the pictures, put the respective names 
- Below the names put the Github and Linkedin icons
-  Use [microanimations](https://useanimations.com/) for the social media icons
- Try your level best to have the images change on click (as in the reference above). 
- The images (max of 3 for each card), text, link to social media should be loaded dynamically from the team.js as the current codebase does
- DO NOT mess up any other part of the web app you are not dealing with

## General PR Guidelines:

- Each pull request should have an appropriate and short title like "Fixed Bug in Upload Page"

- Describe your intended changes in the description section of the pull request (Use bullet points and phrases)

- Refrain from using phrases like "Hi, I am ..", "Please merge me this OPR", "Thank You..", etc. We are only interested in technical parts

- Attach a screenshot/clip of the change(s)

- Make sure to refer to the respective issue in the respective PR using phrases like ``Resolves #issue_number or Closes #issue_number``.

- Please be patient enough. The project maintainers/mentors would review it as per their schedule. Please do not start putting comments like "Please check this" etc.

- Although we support feedback from everyone in all phases of development, it is highly advised not to put any negative comments in other participant's pull requests.

- Always keep a note of the deadline.

### PR DESCRIPTION
**Pull request by ashuvssut**
closes #868 

image

### CODE REVIEW

File: src/pages/Home/sections/Team/Team.jsx [hunk 1]
```diff
@@ -1,95 +1,78 @@
...
-import smaranjitLarge from '../../../../assets/images/team/smaranjit_ghose_large.webp';
-import anushLarge from '../../../../assets/images/team/anush_bhatia_large.webp';
-import smaranjitSmall from "../../../../assets/images/team/smaranjit_ghose_small.webp";
-import anushSmall from "../../../../assets/images/team/anush_bhatia_small.webp";
-
+import React, { useState, useEffect } from "react";
+import styles from "./team.module.scss";
+import { FaGithub } from "react-icons/fa";
+import smaranjit from "../../../../assets/images/team/smaranjit_ghose_small.webp";
+import anush from "../../../../assets/images/team/anush_bhatia_small.webp";
```
Comment (ashuvssut) [inline src/pages/Home/sections/Team/Team.jsx:5]: import maintainers images here

File: src/pages/Home/sections/Team/Team.jsx [hunk 2]
```diff
@@ -1,95 +1,78 @@
...
+import FluidCard from "../../../../components/FluidCard/FluidCard";
 
 function Team() {
+  const maintainerSection = [
+    {
+      name: "Smaranjit Ghose",
+      title: "Lead Developer",
+      githubLink: "https://github.com/smaranjitghose",
+      linkedinLink: "https://www.linkedin.com/in/smaranjitghose/",
+      imgArray: [smaranjit, anush],
```
Comment (ashuvssut) [inline src/pages/Home/sections/Team/Team.jsx:15]: the add that imported image to this imgArray array

**Review by ashuvssut — COMMENTED** [review]
.

**Comment by ashuvssut** [comment]
Remember to resize those maintainer images width: 300 px and height: 400px

**Comment by smaranjitghose** [comment]
> closes #868
> 
> **Pending:** maintainer's images of **proper width(300px) and height(400px)** to be added to assets. Can be done later
> 
> image
> 
> NOTE: the images shown above are placeholder images of wrong sizes

The picture should occupy the entire card. The GitHub and LinkedIn icons should be below the card

**Comment by smaranjitghose** [comment]
For now, for each of our cards, use the same image thrice. Don't swap them. For mine, take the current one from github and the one on my LinkedIn. I believe the same would work for Anush as well

**Comment by ashuvssut** [comment]
in upcoming next PR

- button scale down effect on hover
- canvas scale up effect on hover

- I will try to **_draw_** dynamic (will move along with that liquid effect) shadows under images in canvas. I am not sure about this. but i will give it a try

**Comment by smaranjitghose** [comment]
Make the cards square.

**Comment by smaranjitghose** [comment]
Bug on scrolling down

**Comment by smaranjitghose** [comment]
# Expected Animation:


https://user-images.githubusercontent.com/46641503/115225484-c1155b00-a12b-11eb-9376-c0bd4ac4c007.mp4



# Our Animation (Slow, Often unresponsive on hover):


https://user-images.githubusercontent.com/46641503/115225546-d4282b00-a12b-11eb-8a96-e1d48449d34d.mp4

**Comment by ashuvssut** [comment]
> Bug on scrolling down

wasnt uploaded properly😅

**Comment by ashuvssut** [comment]
@smaranjitghose bhaiya... 

I guess I did this wrong coz I should have aimed to create a repelling like effect. I have to think about how can I achieve that. I ll update after I try to implement that. will take some time

**Comment by ashuvssut** [comment]
render2

**Comment by ashuvssut** [comment]
when the mouse enters the image area.. the repulsion is over. repulsion = 0

This is the same effect as to that of happy monday's website
image

see.. after the mouse enter the canvas image area... the cursor changes to a face

**Comment by smaranjitghose** [comment]
> when the mouse enters the image area.. the repulsion is over. repulsion = 0
> 
> This is the same effect as to that of hellomonday's website
> image
> 
> see.. after the mouse enter the canvas image area... the cursor changes to a face and there is no repulsion
> 
> In our case... the cursor changes pointer cursor. We need someone who can do that cursor face changes

@anushbhatia any suggestions

**Pull request merged**